pkg/operation/botanist/secrets.go

```diff
@@ -323,9 +323,7 @@ func (b *Botanist) GenerateAndSaveSecrets(ctx context.Context) error {
 			"kube-controller-manager",
 			"cluster-autoscaler",
 			"kube-state-metrics",
-			// TODO(rfranzke): Uncomment this in a future release once all monitoring configurations of extensions have been
```

Comment:
Question: this TODO is dropped because only the mountpoint is relevant but not the secret itself. Is that correct?

Reply:
Yes, exactly :) We can already remove the secret from `ShootState` and later we can drop it entirely.